### PR TITLE
Removing unnecessary code which is not used anymore

### PR DESCRIPTION
**Pull request by cockochan**
Thank you for contributing! To help the review process, please provide the following:

### Proposal
Removing unnecessary code and add 'skiped' style to results table and colors from CYF design guide

### Related
Provide links to any associated pull requests or issues. You can use `#123` to link to a PR or issue in this repository, or `user/repo#123` to link to other repositories.

### Checklist

- [ ] I have read the [code of conduct] and [contributing guide]
- [ ] I have made this pull request to the `master` branch
- [ ] I have run all of the automated validation using `npm run ship`
- [ ] I have added myself to the `"contributors"` list in the `package.json` (or do not want to)

[code of conduct]: https://github.com/textbook/starter-kit/blob/master/.github/CODE_OF_CONDUCT.md
[contributing guide]: https://github.com/textbook/starter-kit/blob/master/.github/CONTRIBUTING.md

### CODE REVIEW

File: client/src/Components/Home.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import Quizpic from "../quiztime.png";
+export default function Home() {
+	return (
+		<div className='container'>
+			<div className='row welcomeContainer'>
+				<div className=" welcomeBig col-6">Q</div>
+				<div className=" welcome col-4">Welcome to QuizzTime</div>
```
Comment (pendletons) [inline client/src/Components/Home.js:8]:
```suggestion
				<div className=" welcome col-4">Welcome to QuizTime</div>
```

File: client/src/Components/Home.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+			</div>
+			{/* <img className='quizzTimeLogo' src={Quizpic} /> */}
```
Comment (pendletons) [inline client/src/Components/Home.js:10]: Should this be deleted?

File: client/src/Components/Questions.js [hunk 3]
```diff
@@ -18,7 +18,6 @@ const Questions = (props) => {
 		timestamp: dayjs().format(),
 		studentName: "",
 	});
-	console.log(props.quizData);
```
Comment (pendletons) [inline client/src/Components/Questions.js:21]: 👍

**Review by pendletons — APPROVED** [review]
Looks good! Just one typo to fix.

File: client/src/Components/Students.js [hunk 4]
```diff
@@ -32,8 +24,8 @@ const Students = (props) => {
 			<input
 				type="text"
 				onChange={changeHandler}
-				placeholder="Enter the code"
-				className="input"
+				placeholder="Enter quiz code"
+				className="studentCodeinput"
```
Comment (naderakhgari) [inline client/src/Components/Students.js:28]: why did you change the class name, I just try to prevent of repeatative code and defined a class name to use it for all text input

**Review by naderakhgari — CHANGES_REQUESTED** [review]

File: client/src/Components/Home.js (hunk 1 above)
Comment (cockochan) [reply client/src/Components/Home.js:8]: Thank you for notice

File: client/src/Components/Home.js (hunk 2 above)
Comment (cockochan) [reply client/src/Components/Home.js:10]: That's a placeholder.

File: client/src/App.css [hunk 5]
```diff
@@ -196,16 +210,17 @@ html {
   align-self: center;
 }
 
-/* .survey-page {
+.surveyPage {
   display: flex;
```
[the remark this answers is not in the document]
Comment (cockochan) [reply client/src/App.css:214]: in React never use '-'

File: client/src/App.css [hunk 6]
```diff
@@ -274,17 +289,44 @@ td {
 }
 
 .corect {
-  background-color: #93ff8e;
+  background-color:#81C784;;
```
[the remark this answers is not in the document]
Comment (cockochan) [reply client/src/App.css:292]: This is exactly CYF color from brand book figma called  'Secondary light'
image

File: client/src/App.js [hunk 7]
```diff
@@ -6,7 +6,7 @@ import Students from "./Components/Students.js";
 import Mentors from "./Components/Mentors.js";
 import NewQuestion from "./Components/NewQuestion.js";
 import Results from "./Components/Results.js";
-
+import Home from "./Components/Home";
```
[the remark this answers is not in the document]
Comment (cockochan) [reply client/src/App.js:9]: I suggest a welcome text, logo, image and an input for name to be moved there from students in the future beyound mvp - login input

File: client/src/Components/Students.js (hunk 4 above)
Comment (cockochan) [reply client/src/Components/Students.js:28]: you could be right here, i did not know you use it and wanted to align input differently probably adding another class for alignment can be a better solution
